bad/source/ast.rs

```diff
@@ -217,11 +33,11 @@ pub struct Global<'ctx> {
 	/// The name of the variable.
 	pub name: Id<'ctx>,
 	/// The declared size of this variable, if it is an array.
-	pub size: Option<(ArraySize<'ctx>, Span)>,
+	pub size: Option<(ArraySize<'ctx>, context::Span)>,
```

Comment:
I suspect Span will be enough of a vocabulary type that it's probably gonna be better to refer to it unprefixed. WDYT?

Reply:
I just had it prefixed because I was trying to force myself to get used to all the rules and the way name resolution works. We can fix it back to being `Span` again.